Cargo.toml

```diff
@@ -2,15 +2,15 @@
 members = [
   "bin/node",
   "bin/faucet",
+  "bin/compile-proto",
   "crates/block-producer",
   "crates/proto",
-  "crates/rpc-proto",
   "crates/rpc",
   "crates/store",
   "crates/utils",
   "crates/test-macro",
 ]
-default-members = ["bin/faucet", "bin/node", "crates/proto", "crates/rpc-proto"]
+default-members = ["bin/faucet", "bin/node", "crates/proto"]
```

Comment:
I'm wondering: do we need to have `default-members` at all? If we don't have it, `cargo build` etc. commands will be applied to all crates, right? Not sure that's a bad thing.

Reply:
Correct; its actually somewhat annoying to have as you always have to include the package details for non-default-members.